antidox/wikiwatcher.py

```diff
@@ -111,4 +116,4 @@ def watcher(event_source, wiki_filter, namespaces_filter, callback):
   namespaces = set([int(ns) for ns in args.namespaces.split(',')])
   client = sseclient.SSEClient(args.url)
 
-  watcher(client, args.wiki_filter, namespaces, log_event)
+  watcher(client, args.wiki_filter, namespaces)
```

Comment:
I think you will want to keep the last log_event parameter here, and `callback(change)` in the watcher function above, so that the tests still work

Reply:
We refactored this because we made the `log_event` function take in the API client objects, so we didn't need to recreate those each time. But you're right, we didn't check the tests, which will be broken by this change. Hmm...

I think we could refactor this further to keep the original callback pattern, by binding the API client objects to the `log_event` function (using `functools.partial`, or doing it manually with a lambda).

Reply:
jetpack told me to take it out